app/Blocks/InsetText/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "inset-text",
+  "version": "1.0.0",
+  "description": "GOV.UK Components Inset-Text Block",
```

Comment:
My last comment was misleading, sorry. Can the description ref include a space rather than a hyphen i.e. `Inset Text`, `Inset-Text`.